cvat-canvas/src/typescript/interactionHandler.ts

```diff
@@ -141,14 +147,15 @@ export class InteractionHandlerImpl implements InteractionHandler {
                         _e.preventDefault();
                         _e.stopPropagation();
                         self.remove();
+                        this.shapesWereUpdated = true;
+                        const shouldRiseEvent = this.shouldRaiseEvent(_e.ctrlKey);
```

Comment:
```suggestion
                        const shouldRaiseEvent = this.shouldRaiseEvent(_e.ctrlKey);
```

Reply:
BTW, why have u decided to create a separate identifier to store it? It is used only once, anyway

Reply:
It was used several times, but then I've changed it. It makes no sense now to use variable for this case, i removed it.